inc_internal/ziti_ctrl.h

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (c) 2019 Netfoundry, Inc.
```

Comment:
moving a file is a 'touch'? needs 2019-2020?

Reply:
missed it in update. fixed